cmd/solution/fork.go

```diff
@@ -285,7 +287,15 @@ func forkFromDisk(sourceDir string, fileSystem afero.Fs, solutionName string, st
 			if dir != nil {
 				dirName = dir.Name
 			}
-			return fmt.Errorf("error forking file %q: %w", filepath.Join(dirName, file.Name), err)
+			path := filepath.Join(dirName, file.Name)
+
+			if err == ErrUnsupportedEncoding {
```

Comment:
I don't think this comparison of errors works, I tried it [here](https://go.dev/play/p/YgY-KwM3LUz)

Regardless, a better approach may be to use a custom error.